core/src/main/java/com/netflix/conductor/core/reconciliation/WorkflowSweeper.java

```diff
@@ -78,20 +83,27 @@ public void sweep(String workflowId) {
             WorkflowContext workflowContext = new WorkflowContext(properties.getAppId());
             WorkflowContext.set(workflowContext);
             LOGGER.debug("Running sweeper for workflow {}", workflowId);
-
+            if (!executionLockService.acquireLock(workflow.getWorkflowId())) {
+                return;
```

Comment:
redundant

Reply:
Mirroring the existing functionality , if we cant acquire the lock we want to end the sweep flow for this workflow id , as there is the potential that another thread has it at this moment in time.

Reply:
sorry, I wanted to refer to the `return` of the release block, as there was the `finally`, but it has already been moved